Assets/Editor/UnitTests/Models/ScheduledEventTest.cs

```diff
@@ -17,6 +17,7 @@ public class ScheduledEventTest
 {
     private Action<ScheduledEvent> callback;
     private bool didRun = false;
+    private int runCount = 0;
```

Comment:
Since you're adding this it would be easy enough to get rid of didRun completely. That could be a separate PR after this is merged if you wish.